src/app/pages/ArticlePage/ArticlePage.jsx

```diff
@@ -113,6 +112,8 @@ const ArticlePage = ({ pageData, mostReadEndpointOverride }) => {
   const embedProviderName = path(['model', 'provider'], embedBlock);
   const isUgcUploader = embedProviderName === 'ugc-uploader';
 
+  const mostReadInitialData = path(['secondaryColumn', 'mostRead'], pageData);
```

Comment:
Due to the changes on my PR, I believe the mostRead data will now just be in pageData, rather than within secondary column
```suggestion
  const { mostRead: mostReadInitialData } = pageData;
```